draft-pda-protocol.md

```diff
@@ -1253,6 +1257,18 @@ but server implementations may also opt out of participating in a PDA task if
 the minimum batch size is too small. This document does not specify how to
 choose minimum batch sizes.
 
+## Differential privacy {#dp}
+
+Optionally, PDA deployments can choose to ensure their output F achieves
+[differential privacy](https://en.wikipedia.org/wiki/Differential_privacy).
```

Comment:
Given that the likely trajectory of this document is an IETF WG draft, it would be better to replace this markdown with a `{{dp}}` reference.

Reply:
Edited this comment.

Reply:
Done, I added a general DP reference in place of the wikipedia link. It discusses central, local, and multi-party DP.